region/lowernorfair/west.json

```diff
@@ -87,10 +94,19 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
-                "SpaceJump",
-                "canUsePowerBombs",
-                {"cost": [
-                  {"heatFrames": 1200}
+                {"and": [
+                  "SpaceJump",
+                  "canUsePowerBombs",
+                  {"cost": [
+                    {"heatFrames": 400}
+                  ]}
+                ]},
+                {"and": [
```

Comment:
??

Reply:
It's the walljump way to get back up to the statue for whatever reason one would want that after being there once in the case of some mod removing SpaceJump as a requirement and god knows what else.